src/config/section/offering.js

```diff
@@ -27,7 +27,6 @@ export default {
       docHelp: 'adminguide/service_offerings.html#compute-and-disk-service-offerings',
       icon: 'cloud',
       permission: ['listServiceOfferings', 'listDomains'],
-      params: { isrecursive: 'true' },
```

Comment:
this is not correct. `isrecursive` must be passed to list offerings for domain and sub domain. `isrecursive=true` is inline with legacy UI
![Screenshot from 2020-06-23 16-14-39](https://user-images.githubusercontent.com/153340/85394897-eae20d80-b56c-11ea-8f75-786422f325fa.png)

Reply:
`isrecursive=true` has a bug with disk/compute offerings

It wont display the parent offerings in the child domain. This bug has to fixed in cloudstack codebase as well

Lets say I have ROOT -> child -> grandchild domains and if I created offering for "child" domain then I cant view that in "grandchild" domain using isrecursive=true

Reply:
@shwstppr 

Lets have I have three domains like "ROOT" -> "ROOT/test1" -> "ROOT/test1/test11"

With `isrecursive=true` i dont see all the disk offering of domain "ROOT/test1" from "ROOT/test1/test11"

![Screenshot 2020-06-23 at 16 14 09](https://user-images.githubusercontent.com/10645273/85415052-267cd780-b56d-11ea-85d2-59eddda6d024.png)

with `isrecursive=false` then I do see all offerings of "ROOT/test1" from "ROOT/test1/test11"

![Screenshot 2020-06-23 at 16 14 27](https://user-images.githubusercontent.com/10645273/85415174-4a401d80-b56d-11ea-8f05-2d78e79c0b1a.png)


The cloudmonkey also shows same

```
(test11) 🐵 > list diskofferings listall=true filter=name,domain isrecursive=true
{
  "count": 5,
  "diskoffering": [
    {
      "name": "Small"
    },
    {
      "name": "Medium"
    },
    {
      "name": "Large"
    },
    {
      "name": "Custom"
    },
    {
      "domain": "/test1/test-11/",
      "name": "test-11only"
    }
  ]
}
```

With isrecursive=false

```
(test11) 🐵 > list diskofferings listall=true filter=name,domain
{
  "count": 8,
  "diskoffering": [
    {
      "name": "Small"
    },
    {
      "name": "Medium"
    },
    {
      "name": "Large"
    },
    {
      "name": "Custom"
    },
    {
      "domain": "/test1/",
      "name": "test1only"
    },
    {
      "domain": "/test1/test-11/",
      "name": "test-11only"
    },
    {
      "domain": "/test1/",
      "name": "test1-and-11"
    },
    {
      "domain": "/",
      "name": "root-test1-test11"
    }
  ]
}
```

Reply:
So either we need to retain isrecursive=true and fix the backend code or detain pass isrecursive

Reply:
With current state I don't think we can show parent domain offerings in this view. It will allow showing editing UI for those offerings although user doesn't have access to edit them.
As far as I remember, legacy UI allow showing only current and subdomain offerings in this view and while all accessible offering in deplovm view where isrecursive as true. In any of these better option would be to add an issue in CloudStack itself. @ravening

Reply:
@shwstppr In the legacy Ui also its a bug and the sub child domain cant see the offerings of the parent domain which is the child of "ROOT" domain.
Anyway I will retain the code here and create an issue in cloudstack